OmsAgent/watcherutil.py

```diff
@@ -223,15 +223,23 @@ def upload_telemetry(self):
                             message = status_data["message"]
 
                             event = self.create_telemetry_event(operation,operation_success,message,"300000")
-                            self._hutil_log("Writing telemetry event: "+event)
                             self.write_waagent_event(event)
```

Comment:
should we be truncating the message sent to waagent as well?  would like to ensure that is not overwhelmed as well. thanks

Reply:
Yes, we could, is it ok to do that ?

Reply:
yeah, we saw that it grew to 600MB for the customer yesterday

Reply:
yes for now please also truncate waagent logging, we want to ensure we do not overload that either.

Reply:
Done, we are now truncating it to 100 for waagent too